lib/rom/mapper/attribute_dsl.rb

```diff
@@ -30,6 +29,30 @@ def initialize(attributes, options)
         @reject_keys = options.fetch(:reject_keys)
       end
 
+      # Redefine the prefix for the following attributes
+      #
+      # @example
+      #
+      #   dsl = AttributeDSL.new([])
+      #   dsl.attribute(:prefix, 'user')
+      #
+      # @api public
+      def prefix(value = nil)
+        @prefix = value
+      end
```

Comment:
...oh and the advantage being that it works as a get-or-set method and would only change the value once